roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -17,7 +17,7 @@
       cp {{ openshift_master_config_dir }}/admin.kubeconfig {{ mktemp.stdout }}/admin.kubeconfig
     changed_when: False
 
-  - name: Check for logging project already exists
+  - name: "Check for logging project already exists"
```

Comment:
Most of the task names were quoted in this role so I just stuck with the convention.